src/views/MessagePage.tsx

```diff
@@ -83,25 +83,29 @@ const useStyles = makeStyles(theme => ({
 }));
 
 const MessagePageImpl: React.FC<RouteComponentProps & { fileStore: IFileStore }> = (props) => {
-  const { history, match: { params }, fileStore: { files, getMessage, currentMessage } } = props
+  const { history, match: { params }, fileStore: { files, getMessage, currentMessage, getFile, file } } = props
   const { messageIndex, fileId } = params as any
 
   const classes = useStyles();
   const [[selectedSegmentIndex, selectedFieldIndex], setSelected] = useState([-1, -1]);
+  const [filename, setFileName] = useState('')
```

Comment:
I don't think `useState` and the below `setFileName` call are necessary.
Instead, the name to be displayed can be computed directly from state.

```js
const filename = (file && file.filename) ? file.filename.split('/').pop() : 'Unknown File'
```

Reply:
Good call @orndorffgrant . Implemented!